dashboard/lib/pd/jot_form/translation.rb

```diff
@@ -164,15 +163,20 @@ def parse_jotform_submission(jotform_submission)
       end
 
       # Strip leading and trailing whitespace from each answer.
-      # For each hash, remove "\\" pairs as well, which are appearing in strings
-      # such as "I\\'m a teacher."
+      # For each datetime hash, turn it into a "mm/dd/yyyy" string.
+      # For each non-datetime hash, also remove "\\" pairs from each string as
+      # well, which are appearing in strings such as "I\\'m a teacher."
       def strip_answer(answer)
         if answer.is_a? String
           answer.strip
         elsif answer.is_a? Array
           answer.map(&:strip)
         elsif answer.is_a? Hash
-          answer.transform_values {|value| value.strip.delete("\\")}
+          if ["month", "day", "year"].all? {|k| answer.key?(k)}
+            answer = ["month", "day", "year"].map {|field| answer[field]}.join("/")
```

Comment:
small improvement: you can use `values_at` https://ruby-doc.org/core-2.4.2/Hash.html#method-i-values_at

Reply:
btw, do JotForm return time field?

Reply:
I see a date but not a time in the JotForm API response.